src/vuestic-theme/vuestic-components/vuestic-button/VuesticButton.demo.vue

```diff
@@ -0,0 +1,269 @@
+<template>
+  <div class="demo-container">
+    <div class="demo-container__item" style="width: 1500px;">
+      <table class="table table-bordered test-table">
+        <tr>
+          <th>Description</th>
+          <th>Vuestic Button</th>
+        </tr>
+        <tr>
+          <td>Types</td>
+          <td>
+            <div class="row">
+              <div class="col-4">
+                <vuestic-button color="success">
+                  Default Button
+                </vuestic-button>
+              </div>
+              <div class="col-4">
+                <vuestic-button outline color="success">
+                  Outline Button
+                </vuestic-button>
+              </div>
+              <div class="col-4">
+                <vuestic-button flat color="success">
+                  Flat Button
+                </vuestic-button>
+              </div>
+            </div>
+          </td>
+        </tr>
+        <tr>
+          <td>
+            Sizes
+          </td>
+          <td>
+            <div class="row">
+              <div class="col-4">
+                <vuestic-button color="success">
+                  Default Button
+                </vuestic-button>
+              </div>
+              <div class="col-4">
+                <vuestic-button large color="success">
+                  Large Button
+                </vuestic-button>
+              </div>
+              <div class="col-4">
+                <vuestic-button small color="success">
+                  Small Button
+                </vuestic-button>
+              </div>
+            </div>
+          </td>
+        </tr>
+        <tr>
+          <td>
+            Colors
+          </td>
+          <td>
+            <div class="row">
```

Comment:
There is very little need to use rows/columns in demos.
Just putting whatever you need in tables/divs is enough.